scripts/_utils/utils.py

```diff
@@ -84,6 +91,54 @@ def verify_mimetype(file_url, mimetype_string, local=False):
         return False
 
 
+def is_ffmpeg_compatible(file_url) -> bool:
```

Comment:
Will this work with a web url or just a filepath?  No docstring on this method so won't be helpful to developers in the future.  Code looks like it needs a filepath but variable name is url?